README.md

```diff
@@ -499,3 +499,14 @@ grep -r -l "smt" . | xargs sed -i 's/smt/my-package/'
 #### How can I create benchmarks for my library?
 
 You can see some examples in the `benchmarks` folder. All you have to do is create a file that exports a function to run your benchmark in that folder, and add that function to the `index.ts` file. The `yarn benchmarks` command can be run with no parameters (it will run all the benchmarks), or you can specify the name of your benchmark file to run just that. When you run the command it will create a `benchmarks/results` folder with your results.
+
+#### I need to use a Merkle Tree to prove the inclusion or exclusion of data elements within a set. Which type of Merkle Tree should I use?
+**Incremental:** Ideal for applications where you frequently add new elements and need to update the tree efficiently.
+***Lean Incremental:** A more memory-efficient version of the incremental Merkle tree, suitable for use cases with memory constraints.
+**Sparse:** Best suited for scenarios where you need to manage a large, mostly empty set of possible elements.
```

Comment:
I think we are missing new lines here, The formatting is a bit off
![image](https://github.com/user-attachments/assets/f36214b1-93ef-40f6-b489-b98fadb2d12b)
```suggestion
**Incremental:** Ideal for applications where you frequently add new elements and need to update the tree efficiently.  
***Lean Incremental:** A more memory-efficient version of the incremental Merkle tree, suitable for use cases with memory constraints.  
**Sparse:** Best suited for scenarios where you need to manage a large, mostly empty set of possible elements.
```

Reply:
Working on it